quickwit/quickwit-config/src/source_config/mod.rs

```diff
@@ -365,6 +370,18 @@ pub struct VecSourceParams {
 #[serde(deny_unknown_fields)]
 pub struct VoidSourceParams;
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, utoipa::ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct NatsSourceParams {
+    /// The subject that the source consumes.
+    pub subject: String,
+    /// The connection URI for NATS.
+    pub address: Vec<String>,
```

Comment:
https://docs.rs/async-nats/0.31.0/async_nats/fn.connect.html#with-vec-but-parse-urls-inside-crateconnect this seems to be straightforward?

Reply:
sounds good